packages/opencensus-propagation-stackdriver/src/index.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as v1API from './v1';
+
+export interface SpanContext {
+  /** Trace ID */
+  traceId: string;
+  /** Span ID */
+  spanId: string;
+  /** Options */
+  options?: number;
+}
+
+/**
+ * An transport and environment neutral API for getting request headers.
+ */
+export interface Getter {
```

Comment:
Maybe we should prefer names that have to do more with headers or propagation. `Getter` seems too generic of an interface name.

Reply:
There is a SpanContext already defined in Core package. Do you any restriction to use it here?